week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,15 +1,27 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+// Math.random() method returns a number between 0 (include) and 1 (exclude) and getNumber function multiply this random number with 10 this random and returns the result   
 function getNumber() {
```

Comment:
Great explanations. so basically, the method will return a number between 0 to 9.999...